src/_locales/de/messages.json

```diff
@@ -149,8 +149,13 @@
     "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
   },
   "optionErrorCorrectionDescr": {
-    "message": "Auch wenn ein QR-Code kaputt ist, kann er noch von Maschinen gelesen werden. Dieser Wert zeigt an wie viel von dem Code fehlen kann. Für mehr Information siehe <a href='https://de.wikipedia.org/wiki/QR-Code#Kapazit%C3%A4t_und_Fehlertoleranz'>diesen Wikipedia Eintrag</a>.",
-    "description": "The description of the error correction option."
+    "message": "!HTML! Auch wenn ein QR-Code kaputt ist, kann er immer noch gescannt werden. Dieser Wert zeigt an wie viel von dem Code fehlen kann. <br> Für mehr Information siehe $URL$.",
```

Comment:
There is no need to use spaces around the `<br>` tag. Some browsers could maybe just do awkward things, so better leave them out.

Or… wait… do we even need this forced line break? Such things may not always look good/destroy layout.
Firefox also does not seem to use these on their settings page.